harness/go/history/history.go

```diff
@@ -183,6 +183,7 @@ func scrubRunSpecificScalars(v interface{}) {
 	case *history.WorkflowTaskStartedEventAttributes:
 		v.Identity = ""
 		v.RequestId = ""
+		v.HistorySizeBytes = 0
```

Comment:
Does this fix for both Java and Go?

Reply:
It disables checking this field, should fix any SDKs that support comparing histories: Go and Java today.